ioctl/client.go

```diff
@@ -293,6 +296,36 @@ func (c *client) QueryAnalyser(reqData interface{}) (*http.Response, error) {
 	return resp, nil
 }
 
+func (c *client) ExportHdwallet(password string) ([]byte, error) {
+	// derive key as "m/44'/304'/account'/change/index"
+	hdWalletConfigFile := c.Config().Wallet + "/hdwallet"
+	if !fileutil.FileExists(hdWalletConfigFile) {
+		return nil, errors.New("run 'ioctl hdwallet create' to create your HDWallet first")
+	}
```

Comment:
a single function for check. it's also used in hdwalletxxx command. note the result message is different in these commands.